code/index.html

```diff
@@ -3,28 +3,147 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bodoni+Moda&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap"
+      rel="stylesheet"
+    />
+    <title>Momento photography</title>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header>
+      <div class="hero-section">
+        <picture>
+          <source
+            srcset="./media/hero-mobile.jpg"
+            media="(max-width: 767px)"
+          />
+          <source
+            srcset=".media/hero-tablet.jpg"
+            media="(max-width: 992px)"
+          />
+          <img class="hero-image" src="./media/hero.jpg" alt="Hero image" />
+          <h1>Momento photography</h1>
+        </picture>
+      </div>
+    </header>
 
-    <!-- Signup form -->
+    <section class="first-section">
+      <h2>Destination wedding photographer.<br />Serving worldwide</h2>
+    </section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <section class="middle-section">
+      
+         <!-- Accordion -->
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+      <div class="accordion">
+   
+        <h2>Frequently Asked Questions (FAQ)</h2>
+
+        <div class="question" id="section1">How do I reserve you for my date?</div>
+        <div class="answer">
+          <p>Date is reserved once we receive your signed contract and the deposit.</p>
+        </div>
+
+        <div class="question" id="section2">Do you provide the digital negatives after the shoot?</div>
+        <div class="answer">
+          <p>All of our packages come with Full Resolution image download.</p>
+        </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+        <div class="question" id="section3">How many images do you deliver on the image download?</div>
+        <div class="answer">
+          <p>I typically deliver 60-80 images per hour of coverage. Keep in mind these numbers may increase or decrease depending on the flow of the day and the number of events/activities needing to be captured.</p>
+        </div>
+
+        <div class="question" id="section4">Do you provide partial day coverage?</div>
+        <div class="answer">
+          <p>I typically do not provide partial day coverage on weekends. On weekends, I only accept clients that require at least 8 hours of coverage.</p>
+        </div>
       </div>
+
+      <img
+        class="qa-photo"
+        src="./media/photo-girl.jpg"
+        alt="girl-photographer"
+      />
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <!-- Signup form -->
+    <form class="form" action="https://httpbin.org/anything" method="POST">
```

Comment:
[Here](https://www.w3schools.com/tags/tag_label.asp) you can read about why we should add `<label>`s to our inputs